docs/introduction/en/storage.md

```diff
@@ -0,0 +1,53 @@
+
+In the Conflux network, each storage entry occupies a space of 64B (B is Bytes, byte), which is also the size of the key/value pair in the world state. It should be noted that the key in the blockchain is generally 256bits long and the value It is also 256bits long (each is 32B long, and the total is 64B long). The deposit required for storage is proportional to the minimum multiple of 64B that can cover all stored items. For each storage entry, the last account that writes to the entry is called the owner of the storage entry. If a storage item is written during the execution of contract C, and a guarantor provides guarantee for it, then C is regarded as the writer of the item and accordingly becomes the owner of the item (see section 7.1 for details). In the world state, during the entire life cycle of a storage item, the owner of the item must lock a fixed amount of CFX as a storage deposit for the storage space. Specifically, for each storage entry with a size of 64B, its owner will be locked by 1/16CFX. For occupying `1KB` space, you will pay `1CFX` as a deposit, and the corresponding formula is as follows:
+
+![](/img/storage-formula.png)
+
+When account α becomes the owner of a stored entry (whether it is created or modified), α should immediately lock 1/16 CFX for the entry. If α has enough balance, then the required deposit will be automatically locked, otherwise if α does not have enough balance, the operation will fail and α cannot create or modify the entry.
+
+When a storage item is deleted from the world state, the corresponding 1/16 CFX deposit will be unlocked and returned to the item owner's balance. If the ownership of a storage item changes, the 1/16 CFX deposit of the old owner is unlocked, and the new owner must lock 1/16 CFX as the deposit at the same time. It is worth mentioning that the deposit refund is "quietly" added to the balance, and there is no transfer transaction available for inquiry.
+
+### Solidity memory management mechanism
+
+According to the [Description](https://solidity-cn.readthedocs.io/zh/develop/introduction-to-smart-contracts.html#index-10) of the Solidity document on its memory management and Conflux [Storage Deposit Mechanism](https://mp.weixin.qq.com/s/KKU3XiJqr_Uz5RdlUyQmtA), we can find that the contract storage needs to be maintained with key and value. In general: the length of key is 256bits, and the length of value is also 256bits. The storage space of a smart contract is organized according to the following table, where {0,1}<sup>256</sup> represents a 256-bit string (there are only two values ​​of 0 or 1 in the bit string), **each key/value pair can be understood as an entry**:
+
+Entry | Key/Address({0,1}<sup>256</sup>)   | value({0,1}<sup>256</sup>)  |
+|  :----:  |  :----:  | :----:  |
+1| 0...00000  | 0 |
+2| 0...00001  | 1 |
+3| 0...00002 | 2 |
+...| ... | ... |
+2<sup>256</sup>| f...fffff| 0 |
+
+Since 256bits=32bytes, the corresponding length of two 256bits is: 32+32=64 (Bytes)
```

Comment:
Again, I think there's some confusion here. The "key" part is actually not stored in storage, only the value part. Each storage slot will store the data (32B, same as Ethereum) plus the owner (20B), together they are stored as 64B. Let me discuss this with Chenxing a bit before we update this part.

Reply:
This part is correct, we can keep it as it is.